gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/proc/ReevaluateDagProc.java

```diff
@@ -134,6 +154,7 @@ private void setStatus(DagManagementStateStore dagManagementStateStore,
     for (Dag.DagNode<JobExecutionPlan> node : dag.getNodes()) {
       if (node.getValue().getId().equals(dagNodeId)) {
         node.getValue().setExecutionStatus(executionStatus);
+        dagManagementStateStore.addDagNodeState(node, getDagId());
```

Comment:
was this merely omitted previously or is there only now a need for it, due to asking `ReevaluateDagProc` to do double-duty of launching remaining multi-jobs in addition to regular reevaluation of completed jobs?

Reply:
i realized i was not storing the dag node after modifying it.
however, i realized after making the change that we do not use execution status from dag node, we always call jobstatusretriever to get it.
that also questions the existence of execution status inside a dag node.